entity-framework/core/providers/sql-server/index.md

```diff
@@ -30,12 +30,21 @@ Install-Package Microsoft.EntityFrameworkCore.SqlServer
 > [!NOTE]
 > The provider references Microsoft.Data.SqlClient (not System.Data.SqlClient). If your project takes a direct dependency on SqlClient, make sure it references the Microsoft.Data.SqlClient package.
 
->[!TIP]
+> [!TIP]
 > The Microsoft.Data.SqlClient package ships more frequently than the EF Core provider. If you would like to take advantage of new features and bug fixes, you can add a direct package reference to the latest version of Microsoft.Data.SqlClient.
 
 > [!WARNING]
 > The async implementation of [Microsoft.Data.SqlClient](https://github.com/dotnet/SqlClient) unfortunately has some known issues (e.g. [#593](https://github.com/dotnet/SqlClient/issues/593), [#601](https://github.com/dotnet/SqlClient/issues/601), and others). If you're seeing unexpected performance problems, try using sync command execution instead, especially when dealing with large text or binary values.
 
+## Usage
+
+Besides the expected `UseSqlServer` you can also use `UseAzureSql` and `UseAzureSynapse` methods to specify you're connecting to Azure SQL or Azure Synapse Analytics specifically, which allows the provide optimizations for these platforms. It's also recommended to use `UseCompatibilityLevel` method to specify the compatibility level so that the generated SQL is compatible and/or uses the latest possible features.
```

Comment:
```suggestion
Starting with EF 9, it's recommended to use `UseAzureSql` and `UseAzureSynapse` to specify that you're connecting to Azure SQL or Azure Synapse Analytics specifically, and `UseSqlServer` to specify that you're connecting to on-premises SQL Server; doing so allows the provider to optimize for and properly support these platforms. It's also recommended to use `UseCompatibilityLevel` method to specify the compatibility level so that the generated SQL is compatible and/or uses the latest possible features.
```

Reply:
We may need to include a small note on how to work around the UseNetTopologySuite problem... (and other plugins)